src/js/components/__tests__/HealthBar-test.js

```diff
@@ -0,0 +1,82 @@
+jest.dontMock('../HealthBar');
+jest.dontMock('../StatusBar');
+
+/* eslint-disable no-unused-vars */
+let React = require('react');
+/* eslint-enable no-unused-vars */
+let ReactDOM = require('react-dom');
+let ReactTestUtils = require('react-addons-test-utils');
+let Tooltip = require('reactjs-components').Tooltip;
+
+let HealthBar = require('../HealthBar');
+let StatusBar = require('../StatusBar');
+
+const testData = {
+  tasksRunning: 3,
+  tasksHealthy: 1,
+  tasksUnhealthy: 1,
+  tasksUnknown: 1,
+  tasksStaged: 1
+};
+
+const testInstanceCount = 4;
+describe('#HealthBar', function () {
+  beforeEach(function () {
+    this.container = document.createElement('div');
+    this.instance = ReactDOM.render(
+      <HealthBar
+        tasks={testData}
+        instancesCount={testInstanceCount} />,
+      this.container
+    );
+  });
+
+  afterEach(function () {
+    ReactDOM.unmountComponentAtNode(this.container);
+  });
+
+  describe('PropTypes', function () {
+    it('should throw an error if no tasks prop is provided', function () {
+      spyOn(console, 'error');
+      this.instance = ReactDOM.render(
+        <HealthBar />,
+        this.container
+      );
+      expect(console.error)
+        .toHaveBeenCalledWith(
+          'Warning: Failed propType: Required prop `tasks` was not' +
```

Comment:
let's just change this to make sure that it was called and was passed a string. maybe react might upgrade and decide to change the wording or something.

in fact, i think just checking it was called is fine too